openwisp_users/accounts/templates/account/email/email_confirmation_message.txt

```diff
@@ -1,7 +0,0 @@
-{% extends "account/email/base_message.txt" %}
-{% load account %}
-{% load i18n %}
-
-{% block content %}{% autoescape off %}{% user_display user as user_display %}{% blocktrans with site_name=current_site.name site_domain=current_site.name %}You're receiving this e-mail because user {{ user_display }} registered this e-mail address to create an account on {{ site_domain }}.
```

Comment:
To confirm this is correct, click on the button below.

And then below let's add the call to action button like "Confirm"